tests/sdk/v2/test_dsl.py

```diff
@@ -595,35 +567,6 @@ def test_python_imports_deps(python_imports, expected_req, raises):
         assert list(parsed) == expected_req
 
 
-class TestRefToMain:
```

Comment:
Reminder: this test should not be removed after all. Just not valid for InlineImport

Reply:
Reminding you about the reminder!